Ruqoyah/More-Recipes
# #152906116 Fix Bug in Signup and Profile Page

#### What does this PR do?
It allows users to be able to see error message when an invalid details is passed to the form

#### Description of Task to be completed?
- ensure uploaded image fit in to the image container without deforming
- install cloudinary-react to have more control of the image

#### How should this be manually tested?
- When the user input an invalid details in the signup form, it will show error message
- Also when user go to profile page, and pass in invalid details, he/she will see error message when the

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/152906116
